packages/client/src/components/SafeTokens/EmptyVault.js

```diff
@@ -0,0 +1,12 @@
+const EmptyVault = () => (
+  <div
+    style={{
+      height: "calc(100vh - 350px)",
+    }}
+    className="is-flex is-justify-content-center is-align-items-center is-flex-direction-column"
+  >
+    <h2>You don't have any Vault</h2>
+    <p className="has-text-grey">Create a new transaction above</p>
```

Comment:
the UI seems not as per this design: https://www.figma.com/file/sKtcz6ytBmNNDNW7XU8ea5/Vessel%3A-User-Experience?node-id=344%3A203543, could you help cross check?

Reply:
Thanks for the check, I for sure overlooked. Will update!